README.md

```diff
@@ -27,142 +27,166 @@ The diagram below illustrates high-level design of each component as well as bas
 
 ![Architecture diagram](./assets/architecture.png)
 
-## Usage
+## Installation
 
-Before you can build and run the Miden node or any of its components, you'll need to make sure you have Rust [installed](https://www.rust-lang.org/tools/install). Miden node v0.2 requires Rust version **1.78** or later.
+The node software can be installed as a Debian package or using Rust's package manager `cargo`.
 
-Depending on the platform, you may need to install additional libraries. For example, on Ubuntu 22.04 the following command ensures that all required libraries are installed.
+Official releases are available as debian packages which can be found under our [releases](https://github.com/0xPolygonMiden/miden-node/releases) page.
 
-```sh
-sudo apt install llvm clang bindgen pkg-config libssl-dev libsqlite3-dev
-```
+Alternatively, the Rust package manager `cargo` can be used to install on non-debian distributions or to compile from source.
 
-### Installing the node
+### Debian package
 
-To install for production use cases, run:
+Debian packages are available and are the fastest way to install the node on Debian based system. Currently only `amd64` architectures are supported.
```

Comment:
spelling: should it be "... on a Debian-based system."?

Also, probably "... only `amd64` architecture is supported."

Reply:
Thanks. I should figure out how to integrate a spell-checker into my editor..